src/display/Display.spec.js

```diff
@@ -1 +1,99 @@
+import Dashboard from '../dashboard/Dashboard.js'
+import Controls from '../controls/Controls.js';
+
+afterEach(cleanup);
+
+describe('The Display component', () => {
+    it('provides an indicator that gate is open by default', () => {
+        const { getByText } = render(<Display/>);
+        getByText(/open/i);
+    });
+
+    it('provides an indicator that gate is unlocked by default', () => {
+        const { getByText } = render(<Display/>);
+        getByText(/unlocked/i);
+    });
+
+    it('displays closed when the closed button is clicked', () => {
+        render(<Dashboard/>);
+        const controls = render(<Controls/>);
+        const closeButton = controls.getByText(/close gate/i);
+        const display = render(<Display/>);
+        const closedIndicator = display.getByText(/open/i);
+
+        fireEvent.click(closeButton);
+
+        expect(closedIndicator).toHaveTextContent(/closed/i);
+    });
+
+    it('displays open when the open button is clicked', () => {
+        render(<Dashboard/>);
+        const controls = render(<Controls/>);
+        const closeButton = controls.getByText(/close gate/i);
+        const display = render(<Display/>);
+        const openIndicator = display.getByText(/open/i);
+
+        fireEvent.click(closeButton);
+        fireEvent.click(closeButton);
+
+        expect(openIndicator).toHaveTextContent(/open/i);
+    });
+
+    it('displays open when the open button is clicked', () => {
+        render(<Dashboard/>);
+        const controls = render(<Controls/>);
+        const closeButton = controls.getByText(/close gate/i);
+        const lockButton = controls.getByText(/lock gate/i);
+        const display = render(<Display/>);
+        const lockedIndicator = display.getByText(/unlocked/i);
+
+        fireEvent.click(closeButton);
+        fireEvent.click(lockButton);
+
+        expect(lockedIndicator).toHaveTextContent(/locked/i);
+    });
+
+    it('displays unlocked when the unlock button is clicked', () => {
+        render(<Dashboard/>);
+        const controls = render(<Controls/>);
+        const closeButton = controls.getByText(/close gate/i);
+        const lockButton = controls.getByText(/lock gate/i);
+        const display = render(<Display/>);
+        const unlockedIndicator = display.getByText(/unlocked/i);
+
+        fireEvent.click(closeButton);
+        fireEvent.click(lockButton);
+        fireEvent.click(lockButton);
+
+        expect(unlockedIndicator).toHaveTextContent(/unlocked/i);
+    });
+
+    it('displays green when gate is open', () => {
+        const display = render(<Display closed={false}/>);
+        const openIndicator = display.getByText(/open/i);
+        expect(openIndicator).toHaveClass('led green-led');
+    });
+
+    it('displays red when gate is closed', () => {
+        const display = render(<Display closed={true}/>);
+        const closedIndicator = display.getByText(/closed/i);
+        expect(closedIndicator).toHaveClass('led red-led');
```

Comment:
I know we left on the note of different ways to implement these test cases but for this project specifically, it is pretty vanilla. One thing I can suggest is for you is to make different changes other than checking what is provided. Experimenting outside of the box!